spec/features/courses/modern_languages/new_spec.rb

```diff
@@ -92,21 +117,23 @@
         new_provider_recruitment_cycle_courses_age_range_path(
           provider_code: provider.provider_code,
           recruitment_cycle_year: recruitment_cycle.year,
-          course: { subjects_ids: [modern_languages_subject.id, russian.id] },
+          course: { subjects_ids: [modern_languages_subject.id] }, # russian is missing because there's no real api to remember the choice
```

Comment:
It's unclear to me what this comment means

Reply:
What I was trying to say is that if you look further up there's a `       new_modern_languages_page.language_checkbox("Russian").click` but confusingly it's not then expected to be in the final url because the whole backend is mocked out. I agree it could be improved. Perhaps we should jump on a call and discuss further to come up with a better explanation or shape of code.